SU-SWS/stanford_magazine
# SOE-2372: Doc and version updates for 7.x-1.0-beta2 release

# READY FOR REVIEW

# Summary
- Release template and associated version updates for push to production.

# Needed By (11/13)
- I hope to stage a 7.x-1.0-beta2 release on jse-soe-dev by Monday, so you can commit it to the gitolite repo.

# Urgency
- Between 1 and 10, perhaps a 7.

# Steps to Test

1. Check out this branch
2. Manually review the diff for anything, checking for anything that was *not* documentation or version related
3. Review the diff with `--name-only` to verify that only `CHANGELOG`, `README`, and `.info` files were modified
4. Recursively grep for `7.x-1.0-beta1  ` to ensure that it was only found in `CHANGELOG`

Also be sure the Release Template below looks accurate:

### What (Changelog)
- Updates default image for featured, stanford_magazine pages.

### So what
- This work was undertaken to improve load times on the site, by using a smaller default image.  And so that we could revert the stanford_magazine Feature on the SOE site.

### Now what
- You’ll need to revert the stanford_magazine feature and update the database for these changes to appear on production.

### New Requirements and Risks (Changelog)
- No known new requirements or risks.

### Diff
- https://github.com/SU-SWS/stanford_magazine/compare/b53a0b1438223b5497f1a22b6857541f3fe3f038...36ec3be42cd8801d8eea9018955ef0d74d2e4d9d

# Affected Projects or Products
- SOE home site

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-2372
- https://stanfordits.atlassian.net/browse/SOE-2354

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
